baseframe/templates/baseframe/forms-material.html

```diff
@@ -0,0 +1,334 @@
+    {%- if field.type in ['CSRFTokenField', 'HiddenField'] -%}
+      {# Don't show hidden #}
+    {%- else -%}
+      {{ renderfield(field, autofocus=autofocus, style=style) }}
+      {%- if autofocus %}{% set autofocus = false %}{% endif %}
+    {%- endif %}
+  {% endfor %}
+{%- endmacro %}
+
+{% macro renderform(form, formid, submit, message='', action='', cancel_url='', multipart=false, style='horiz') %}
+<form data-parsley-validate id="{{ formid }}" method="POST" {%- if action %} action="{{ action }}" {%- endif %}{%- if multipart %} enctype="multipart/form-data" {%- endif %} accept-charset="UTF-8" class="mui-form mui-form--margins">
+  {{ renderform_inner(form, formid or none, style=style) }}
+  {{ rendersubmit([(none, submit or _("Submit"), 'mui-btn--primary')], cancel_url=cancel_url, style=style, csrf_error=form.csrf_token.errors) }}
+</form>
+{% endmacro %}
+
+{%- macro ajaxform(formid, request, force=false) %}
+  {%- if force or request.is_xhr %}
+    <script type="text/javascript">
+      $(function() {
+        $("#{{ formid }}").ajaxForm({
+          target: '#{{ formid }}',
+          replaceTarget: true,
+          beforeSubmit: function(formdata, form, options) {
+            form.find('input[type="submit"]').prop('disabled', true).addClass('submit-disabled');
+            form.find('button[type="submit"]').prop('disabled', true).addClass('submit-disabled');
+            form.find('.loading').removeClass('hidden');
+            return true;
+          },
+          error: function(context, xhr, status, errMsg) {
+            var form = $("#{{ formid }}");
+            form.find('input[type="submit"]').prop('disabled', false).removeClass('submit-disabled');
+            form.find('button[type="submit"]').prop('disabled', false).removeClass('submit-disabled');
+            form.find('.loading').addClass('hidden');
+            form.append('<div class="alert alert--danger"><a class="alert__close" href="javascript:void(0);" aria-label="close"><i class="material-icons">clear</i></a><p class="alert__text"><i class="material-icons alert__icon">error</i> An error occured when submitting the form</p></div>');
+          } // No comma or semicolon here
+        });
+      });
+    </script>
+  {%- else %}
+    <script type="text/javascript">
+      $(function() {
+        // Disable submit button when clicked. Prevent double click.
+        $('#{{ formid }} input[type="submit"]').click(function() {
+          $(this).addClass('submit-disabled').prop('disabled', true);
+          $('#{{ formid }}').find(".loading").removeClass('hidden');
+          this.form.submit();
+        });
+        $('#{{ formid }} button[type="submit"]').click(function() {
+          $(this).addClass('submit-disabled').prop('disabled', true);
+          $('#{{ formid }}').find(".loading").removeClass('hidden');
+          this.form.submit();
+        });
+      });
+    </script>
+  {%- endif %}
+{%- endmacro %}
+
+{% macro tinymce3_editor(field, script=true) -%}
```

Comment:
We can drop support for TinyMCE3 in Baseframe-MUI.

Reply:
Removed TinyMCE3 in Baseframe-MUI template
Updated hasgeektv to use TinyMCE4